### ISSUE TITLE
Q：More effective hyperparameter settings

### ISSUE THREAD

**Issue by cn-xvkong** [ShashankHuang/RecLMIS#7]
Hello! Thank you for your outstanding work. I would like to know more about possible effective hyperparameter settings, including the initial settings of the optimizer and learning rate, as well as the learning rate adjustment strategy. I am looking forward to your response!
